### PR TITLE
Selenium: add selenium tests to UNDER_REPAIR group

### PR DESCRIPTION
**Pull request by SkorikSergey**
### What does this PR do?
This PR:
- adds **AutocompleteProposalJavaDocTest** selenium test to UNDER_REPAIR group;
- add checking ```angular-patternfly-starter``` sample project  in **CreateWorkspaceFromNodeStackTest** selenium test to UNDER_REPAIR group(https://github.com/eclipse/che/issues/12259).

### CODE REVIEW

**Comment by SkorikSergey** [comment]
ci-build

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/editor/autocomplete/AutocompleteProposalJavaDocTest.java [hunk 1]
```diff
@@ -39,6 +39,7 @@
 import org.testng.annotations.Test;
 
 /** @author Dmytro Nochevnov */
+@Test(groups = UNDER_REPAIR)
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/editor/autocomplete/AutocompleteProposalJavaDocTest.java:42]: UNDER_REPAIR group at the test method level looks redundant then.

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/stack/CreateWorkspaceFromCentosNodeStackTest.java [hunk 2]
```diff
@@ -81,7 +82,7 @@ public void checkWorkspaceCreationFromCentosNodeStack() {
     projectExplorer.waitProjectInitialization(WEB_NODE_JS_PROJECT);
   }
 
-  @Test(priority = 1)
+  @Test(priority = 1, groups = UNDER_REPAIR)
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/stack/CreateWorkspaceFromCentosNodeStackTest.java:87]: It's expected that test under repair points to issue which describe permanent test failure, in try-catch block around failing test command.

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/stack/CreateWorkspaceFromNodeStackTest.java [hunk 3]
```diff
@@ -86,7 +87,7 @@ public void checkWorkspaceCreationFromNodeStack() {
     projectExplorer.waitProjectInitialization(WEB_NODE_JS_PROJECT);
   }
 
-  @Test(priority = 1)
+  @Test(priority = 1, groups = UNDER_REPAIR)
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/stack/CreateWorkspaceFromNodeStackTest.java:90]: It's expected that test under repair points to issue which describe **permanent** test failure, in try-catch block around failing test command.

**Comment by SkorikSergey** [comment]
ci-build

**Review by dmytro-ndp — APPROVED** [review]

**Pull request merged**